getAccessToken.js

```diff
@@ -0,0 +1,14 @@
+const { getAccessTokenFromApiKey } = require('./utils/apiUtil');
+
+TARGET_ENVIRONMENT = process.env.GEN3_COMMONS_HOSTNAME;
+// console.log('target environment: ' + TARGET_ENVIRONMENT);
+API_KEY = process.env.API_KEY;
+// console.log('api key: ' + API_KEY);
+
```

Comment:
Maybe add a comment to point out that command line argument processing is handled in `getAccessToken.sh` ...

Reply:
Good call. Pushing the change now.